libs/ui-components/src/index.ts

```diff
@@ -1,4 +1 @@
-import Icon from "./ui-atoms/Icon";
-import { ChainSwitcher } from "./components/ChainSwitcher";
-import { ConnectButton } from "./components/ConnectButton";
-export { Icon, ChainSwitcher, ConnectButton };
+export * from './ui-atoms';
```

Comment:
this is not correct since:
import { ChainSwitcher } from "./components/ChainSwitcher";
import { ConnectButton } from "./components/ConnectButton";
are not inside ui-atoms
Drop every edit inside libs/ui-components since they are not relevant for nx migration